frontend/packages/client/src/hooks/useCommunity.js

```diff
@@ -56,13 +60,15 @@ export default function useCommunity({
       try {
         const timestamp = Date.now().toString();
         const hexTime = Buffer.from(timestamp).toString('hex');
-        const _compositeSignatures = await injectedProvider
-          .currentUser()
-          .signUserMessage(hexTime);
 
-        const compositeSignatures = getCompositeSigs(_compositeSignatures);
+        const [compositeSignatures, voucher] =
+          await signMessageByWalletProvider(
+            user?.services[0]?.uid,
+            CREATE_COMMUNITY_TX,
+            hexTime
+          );
```

Comment:
thats a good idea, then we can include additional data for this arg as well, and just always append `hexTime` as one of the args automatically

Reply:
i'm putting this off for now until it is needed (i suspect it will be, because there may be another param we need to pass in) but i did consolidate all the `timestamp/hexTime` lines into a single line of code.